### ISSUE TITLE
Zooey unable to process 3 staged battles that ended on turn 1

### ISSUE THREAD

**Issue by ErogeSeiyuu** [Masuzu/ZooeyBot#78]
This is the script I'm testing for, tailored to finish Extreme battle quickly.
```
if battle == 1 and turn == 1 then
characters["Zooey"]:UseSkill(2)
characters["Zooey"]:UseSkill(3):WithWaitTime(10000)
SkipAttack()
end
if battle == 2 and turn == 1 then
characters["Orchid"]:UseSkill(1)
characters["Orchid"]:UseSkill(3):WithWaitTime(10000)
SkipAttack()
end
if battle == 3 and turn == 1 then
characters["Zooey"]:UseSkill(1)
end
```

The bot crashes whenever s.zoi uses her 3 (which instantly kill all mobs on battle 1) and move onto the next stage. 

My ZooeyBot.ini did not change anything from its original state aside from UseViramate=true and LuaScripts=Scripts/Test.lua

I think the issue is that Zooey cannot recognize turns that ended without a turn passing. S.zoi's 3 and Orchid's 3 can kill all mobs in the room without making a turn pass, so it's the reason Zooey unable to recognize the change in battle phase and crash.

**Comment by Masuzu** [comment]
Hi,
On it and trying to reproduce the situation. I remember about a recent change which may cause the crash and will look into it.

**Comment by Masuzu** [comment]
Ah ok, the issue was really silly.
Somehow SkipAttack was causing the crash as it was parsed as a "skill".
For now please do not use this command unless you are using the slave mode while I patch it.

**Comment by Masuzu** [comment]
This is now fixed in version 1.6.4:
https://poker-bot.azurewebsites.net/ZooeyBot/en/Home/PatchNotes
